ci/jenkins/test.sh

```diff
@@ -327,12 +327,15 @@ function prepare_env {
 function revert_snapshot_windows {
     WIN_NAME=$1
     echo "==== Reverting Windows VM ${WIN_NAME} ====="
-    if [[ $WIN_NAME == *"jumper"* ]]; then
-        govc snapshot.revert -vm ${WIN_NAME} win-initial
+    if [[ ${TESTCASE} =~ "containerd" ]]; then
+        if [[ $WIN_NAME == *"jumper"* || $WIN_NAME == *"win-image"* || ${TESTCASE} == *"userspace"*]]; then
```

Comment:
if TESTCASE contains userspace will it use win-initial snapshot or pristine ?

Reply:
win-initial